shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -36,8 +35,9 @@ const strToString = "toString";
 const strConstructor = "constructor";
  
 const _objDefineProperty = ObjDefineProperty;
-const _objFreeze = ObjClass["freeze"];
-const _objSeal = ObjClass["seal"];
+const _objFreeze = ObjClass.freeze;
+const _objSeal = ObjClass.seal;
+const _objKeys = ObjClass.keys;
```

Comment:
These are all static functions and using ["name"] was 4 more characters than really required. Used that notation previously because of the version of typescript we used. (v2 from memory)